rskj-core/src/main/java/org/ethereum/json/JSONHelper.java

```diff
@@ -109,10 +110,10 @@ public static void dumpBlock(ObjectNode blockNode, Block block,
 
         ObjectNode statesNode = blockNode.objectNode();
         for (ByteArrayWrapper key : keys) {
-            byte[] keyBytes = key.getData();
-            AccountState accountState = repository.getAccountState(keyBytes);
-            ContractDetails details = repository.getContractDetails(keyBytes);
-            dumpState(statesNode, Hex.toHexString(keyBytes), accountState, details);
+            RskAddress addr = new RskAddress(key.getData());
```

Comment:
I think this complete method is unused

Reply:
It's actually the whole class. I'll get rid of it.